packages/remme-utils/src/models/index.ts

```diff
@@ -28,6 +29,15 @@ export class PublicKeyRequest {
     ) {}
 }
 
+/**
+ * Model that define node account address params into request
+ */
+export class NodeAccountAddressRequest {
```

Comment:
why this in utils? This payload in use only in one place and this place in remme-node-management, pls move this to it